PsfLauncher/main.cpp

```diff
@@ -512,7 +375,7 @@ void LogString(const char name[], const wchar_t value[]) noexcept
     Log("\t%s=%ls\n", name, value);
 }
 
-ErrorInformation CheckIfPowershellIsInstalled(bool& isPowershellInstalled) noexcept
+void CheckIfPowershellIsInstalled(bool& isPowershellInstalled)

```

Comment:
>bool& isPowershellInstalled [](start = 34, length = 27)

Now that this function uses exceptions, you can just return this bool instead of using a gross out-param.

Reply:
I like it, changed to return bool

---
In reply to: [313159157](https://github.com/microsoft/MSIX-PackageSupportFramework/pull/77#discussion_r313159157) [](ancestors = 313159157)